src/strings/PowerFxResources.en-US.resx

```diff
@@ -4760,4 +4760,12 @@
     <value>Unsupported untyped/JSON conversion type '{0}' in argument.</value>
     <comment>Error Message shown to user when a unsupported type is passed in type argument of AsType, IsType and ParseJSON functions.</comment>
   </data>
+  <data name="ErrReachedMaxJsonDepth" xml:space="preserve">
+    <value>Maximum depth reached while traversing JSON payload.</value>
+    <comment>Error message returned by the {Locked=JSON} function when a document that is too deeply nested is passed to it. The term JSON refers to the data format described in www.json.org.</comment>
```

Comment:
>{Locked=JSON} [](http://example.com/codeflow?start=43&length=13)

In this case we _don't_ want to lock the term JSON. It doesn't refer to the function, but to the format itself. If the term is translated in some language, we want the translators to be able to do that.